manticore/native/state.py

```diff
@@ -17,6 +35,21 @@ def mem(self):
         """
         return self._platform.current.memory
 
+    def _checkpoint(self) -> CheckpointData:
+        """
+        Checkpoint all necessary information in the case of a rollback.
+        """
+        self._checkpoint_data = CheckpointData(pc=self.cpu.PC, last_pc=self.cpu._last_pc)
+        return self._checkpoint_data
+
+    def rollback(self, checkpoint_data: CheckpointData) -> None:
```

Comment:
It looks like the `checkpoint_data` argument that is supplied at every use site is `self._checkpoint_data`. If this is indeed the case, why not eliminate the `checkpoint_data` parameter here, and have the implementation of this function always use `self._checkpoint_data` instead?

Reply:
I was thinking it might be useful if we ever wanted to restore from another state's `checkpoint_data` 🤷 

Probably best not to do that until it's needed though.